telebot/__init__.py

```diff
@@ -185,6 +191,12 @@ def __init__(
         self.allow_sending_without_reply = allow_sending_without_reply
         self.webhook_listener = None
         self._user = None
+        self.bot_id: int = None
+
+        if validate_token:
+            util.validate_token(self.token)
+
+        self.bot_id = util.extract_bot_id(self.token) # subject to change in future, unspecified
```

Comment:
May cause error if token is not in valid form. extract_bot_id should return NONE if extract fails.

Reply:
But validate_token raises an error if token is not in valid form?

Reply:
Yes. But only if validate_token is enabled. It's optional.

Reply:
oh yes. Btw, you are fine with validate_token being True by default?

Reply:
I saw it. I'm not happy, but as far it is not breaking and can be disabled - I can deal with it.